primitives/src/ztg.rs

```diff
@@ -0,0 +1,39 @@
+// Distribution
```

Comment:
Taken from the website.

The only necessary parameters here are `PARACHAIN_LEASE`, `CROWDLOAN`, `LIQUIDITY_MINING` and `STAKING`. Everything else is surplus but will eventually be used in the future.